randovania/games/dread/json_data/Itorash.txt

```diff
@@ -205,11 +205,26 @@ Extra - asset_id: collision_camera_004
 
 > Boss - Raven Beak; Heals? False
   * Layers: default
-  > Dock to Hanubia
+  > Event - Escape Sequence
+      All of the following:
+          Lay Power Bomb
+          Any of the following:
+              Space Jump or Simple IBJ
+              All of the following:
+                  Speed Booster and Spin Boost
+                  Movement (Intermediate) or Speed Booster Conservation (Beginner)
+                  Single-wall Wall Jump (Intermediate) or Wall Jump (Beginner)
```

Comment:
I'm not too sure what the single-wall wall jump/wall jump is for. Why is this here?